.sync.yml

```diff
@@ -24,6 +24,13 @@ Gemfile:
       - gem: master_manipulator
       - gem: puppet-blacksmith
         version: '~> 3.4'
+      - gem: 'puppet_litmus'
```

Comment:
has pdksync been run recently ? 
the gem does not need to be added anymore, as it is in puppet-module-gems

Reply:
How many places do we want to run the acceptance tests? They are already running in Appveyor, and again in Beaker. Do we want Puppet Pipelines runs in addition to all that?

Reply:
Removed reference to litmus gem